.pre-commit-config.yaml

```diff
@@ -0,0 +1,40 @@
+ci:
+  autofix_prs: false
+repos:
+  - repo: https://github.com/asottile/reorder_python_imports
+    rev: v3.0.1
+    hooks:
+      - id: reorder-python-imports
+        language_version: python3
```

Comment:
it makes sense to use the defaut language version setting for python (as its used everywhere)